AK/FixedPoint.h

```diff
@@ -219,7 +219,7 @@ class FixedPoint {
         // If last bit cut off is 1:
         if (value & (1u << (precision - 1))) {
             // If the bit after is 1 as well
```

Comment:
This should probably say `any bit after`, but is not that big of an issue right now